state-chain/pallets/cf-elections/src/electoral_systems/composite.rs

```diff
@@ -575,3 +576,4 @@ generate_electoral_system_tuple_impls!(tuple_1_impls: ((A, A0)));
 generate_electoral_system_tuple_impls!(tuple_2_impls: ((A, A0), (B, B0)));
 generate_electoral_system_tuple_impls!(tuple_3_impls: ((A, A0), (B, B0), (C, C0)));
 generate_electoral_system_tuple_impls!(tuple_4_impls: ((A, A0), (B, B0), (C, C0), (D, D0)));
+generate_electoral_system_tuple_impls!(tuple_5_impls: ((A, A0), (B, B0), (C, C0), (D, D0), (EE, E0)));
```

Comment:
`EE` because `E` conflicts with the E generic as in Error... I might make them all double letters for consistency.... but not super important

Reply:
Or since they aren't intended to be used, we could make that clear by prefixing them all? `_A, _B, _C` etc ?

Reply:
Yeah that probably works better

Reply:
I will do this once in main just to avoid more rounds of chained rebasing